src/components/Hero/Hero.stories.tsx

```diff
@@ -189,20 +206,42 @@ export const WithControls: Story = {
 // The following are additional Accordion example Stories.
 export const Primary: Story = {
   render: () => (
-    <Hero
-      backgroundImageSrc="//placekitten.com/1600/800"
-      heading={
-        <Heading
-          id="primary-hero"
-          level="h1"
-          noSpace
-          overline="Hero Example"
-          subtitle="Integer posuere erat a ante venenatis dapibus posuere velit aliquet."
-          text="Hero Primary"
-        />
-      }
-      heroType="primary"
+    <>
+      <Hero
+        backgroundColor="ui.white"
+        backgroundImageSrc="//loremflickr.com/1600/800/kitten"
+        heading={
+          <Heading
+            id="primary-hero"
+            level="h1"
+            noSpace
+            overline="Hero Example"
+            subtitle="Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Vestibulum id ligula porta felis euismod semper. Aenean eu leo quam."
+            text="Hero Primary"
+          />
+        }
+        heroType="primary"
+      />
+      <Source
+        code={`
+<Hero
+  backgroundImageSrc="//loremflickr.com/1600/800/kitten"
+  heading={
```

Comment:
Missing `backgroundColor="ui.white"` here. Also, this looks off...
![Screen Shot 2024-04-05 at 11 53 48 AM](https://github.com/NYPL/nypl-design-system/assets/1280564/716f7ea6-ec3e-471f-a94a-f12802755a68)

Reply:
The inclusion of `backgroundColor="ui.white"` was a mistake, so I just remove it.